guava/src/module-info.java

```diff
@@ -0,0 +1,45 @@
+// GENERATED FILE - DO NOT EDIT
+
+/*
+ * Copyright (C) 2008 The Guava Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Google Guava
+ */
+module com.google.common {
+  requires java.base;
+  requires java.logging;
+  requires jdk.unsupported;
+  requires com.google.common.util.concurrent.internal;
+
+  exports com.google.common.annotations;
+  exports com.google.common.base;
+  exports com.google.common.cache;
+  exports com.google.common.collect;
+  exports com.google.common.escape;
+  exports com.google.common.eventbus;
+  exports com.google.common.graph;
+  exports com.google.common.hash;
+  exports com.google.common.html;
+  exports com.google.common.io;
+  exports com.google.common.math;
+  exports com.google.common.net;
+  exports com.google.common.primitives;
+  exports com.google.common.reflect;
+  exports com.google.common.util.concurrent;
+  exports com.google.common.xml;
+  exports com.google.thirdparty.publicsuffix;
```

Comment:
Should `com.google.thirdparty.publicsuffix` actually be exported?
All the public classes in this package state the following, which makes it seems like they are meant for internal use only:
```
Do not use this class directly. For access to public-suffix information, use com.google.common.net.InternetDomainName.
```

Reply:
Thanks, yes, we should have put those in `common.net` itself so that we could have made them package-private. (We may still move them someday, but it requires untangling some things in our internal builds.) It would be better to not export them.

Reply:
Easy fix, thanks for spotting